src/LessMsi.Gui/MainForm.cs

```diff
@@ -722,13 +736,13 @@ private void InitializeComponent()
             this.exitToolStripMenuItem});
             this.fileToolStripMenuItem.Name = "fileToolStripMenuItem";
             this.fileToolStripMenuItem.Size = new System.Drawing.Size(37, 20);
-            this.fileToolStripMenuItem.Text = "&File";
+            this.fileToolStripMenuItem.Text = $"&{Strings.File}";
             // 
             // openToolStripMenuItem
             // 
             this.openToolStripMenuItem.Name = "openToolStripMenuItem";
             this.openToolStripMenuItem.Size = new System.Drawing.Size(152, 22);
-            this.openToolStripMenuItem.Text = "&Open";
+            this.openToolStripMenuItem.Text = $"&{Strings.Open}";

```

Comment:
Same as above - put the `&` character into the translations themsleves incase a different keyboard accelerator makes more sense in other languages.